modules/google-analytics/readme.txt

```diff
@@ -0,0 +1,133 @@
+=== WP Google Analytics ===
```

Comment:
We'll need to remove that file from the plugin, to avoid issues when deleting the Jetpack plugin.